find all numbers dissappeared

```diff
@@ -0,0 +1,19 @@
+class Solution {
+    public List<Integer> findDisappearedNumbers(int[] nums) {
+        List<Integer> list = new ArrayList<Integer>();
```

Comment:
Good Job! It would be great if you can add few comment in your code explaining your approach.